templates/tutorial.html

```diff
@@ -45,6 +45,7 @@ <h2>
       </div>
     </div>  
 
+		$if(lts)$
```

Comment:
@Larox The indentation looks weird here. Could you please make sure it matches the rest of the file?

Reply:
Good catch! Updating!